pod/perl5122delta.pod

```diff
@@ -122,7 +122,7 @@ role in the exception mechanism.
 
 =item *
 
-Perl's L<INSTALL> file has been clarified to explicitly state that Perl
+Perl's F<INSTALL> file has been clarified to explicitly state that Perl
 requires a C89 compliant ANSI C Compiler.
```

Comment:
I know this isn't part of your patch, but Isn't perl C99 now days?

Reply:
Yes (partially), but this is perl5122delta, which was released 15 years ago.